pkg/handler/v2/routes_v2.go

```diff
@@ -6787,3 +6797,61 @@ func (r Routing) deleteResourceQuota() http.Handler {
 		r.defaultServerOptions()...,
 	)
 }
+
+//swagger:route get /api/v2/projects/{project_id}/groupbindings groupProjectBinding listGroupProjectBinding
```

Comment:
maybe keep this under project tag?
```suggestion
//swagger:route get /api/v2/projects/{project_id}/groupbindings project listGroupProjectBinding
```